frontend/flashfolio/src/Viewer.js

```diff
@@ -107,6 +131,8 @@ export default function Viewer({viewMode="view"}) {
 			<button id= "shuf" onClick = {() => shufFunction()}>Shuffle</button> :
 			<button id="unshuf" onClick = {() => shufOn ? unshufFunction() : null}>Unshuffle</button>)
 		}
+		<button onClick={deleteCard}>
```

Comment:
Please make sure that button is only visible when `viewMode === "edit"`

We don't want user to be able to delete cards when they're just viewing a deck.

Reply:
i have made this change